lib/stellar_core_commander/docker_process.rb

```diff
@@ -216,14 +226,12 @@ def config
 
         QUORUM_THRESHOLD=#{threshold}
 
-        PREFERRED_PEERS=#{peers}
+        PREFERRED_PEERS=#{peer_connections}
         QUORUM_SET=#{quorum}
 
-        HISTORY_PEERS=["main"]
-
-        HISTORY_GET=cp history/%s/{0} {1}
-        HISTORY_PUT=cp {0} history/%s/{1}
-        HISTORY_MKDIR=mkdir -p history/%s/{0}
+        HISTORY_PEERS=#{peer_names}
```

Comment:
Should be pretty easy to make this conditional, where we fallback to the local history if the s3 cli arguments are blank, right?